app/views/api/v2/users/main.json.rabl

```diff
@@ -4,6 +4,8 @@ extends "api/v2/users/base"
 
 attributes :firstname, :lastname, :mail, :admin, :auth_source_id, :auth_source_name, :timezone, :locale, :last_login_on, :created_at, :updated_at
 
+node(:admin) { |user| user.admin? }
```

Comment:
Naming this attribute `admin` will override the line above, hiding whether the user has the admin flag or not.  It won't be possible to differentiate between a user the admin flag or a user who's inherited it.

The UI does differentiate in part, by showing the inherited admin status on the list but the edit form would show the value of the user's attribute.

Perhaps `effective_admin` or a separate flag to show only whether admin is inherited also from a group, e.g. `inherits_admin`.